database-migration/src/main/java/uk/ac/ebi/eva/dbmigration/mongodb/ExtractAnnotationFromVariant.java

```diff
@@ -345,4 +348,20 @@ public void createIndexes(MongoDatabase mongoDatabase) {
         annotationsCollection.createIndex(new Document(CHROMOSOME_FIELD, 1).append(START_FIELD, 1).append(END_FIELD, 1),
                                           background);
     }
+    @ChangeSet(order = "006", id = "addDefaultVersionInAnnotationMetadata", author = "EVA")
+    public void addDefaultVersion(MongoDatabase mongoDatabase) {
+        final MongoCollection<Document> annotationMetadataCollection = mongoDatabase.getCollection(
+                databaseParameters.getDbCollectionsAnnotationMetadataName());
+        logger.info("6) add default annotation version to collection {} ", annotationMetadataCollection.getNamespace());
+
+        Document all = new Document();
+        Document versionIsNotTheDefaultOne = new Document("$set", new Document(DEFAULT_VERSION_FIELD, false));
+        annotationMetadataCollection.updateMany(all, versionIsNotTheDefaultOne);
+
+        String id = databaseParameters.getVepVersion() + "_" + databaseParameters.getVepCacheVersion();
+        Document defaultVersionDocument = new Document(ID_FIELD, id);
+        Document setDefaultToTrue = new Document("$set", new Document(DEFAULT_VERSION_FIELD, true));
```

Comment:
The negative case is named `versionIsNotTheDefaultOne` and the positive `setDefaultToTrue`. Try to make them a bit more homogeneous.